src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -201,7 +208,8 @@ object SnowflakeTelemetry {
                                          queryStatus: String,
                                          elapse: Long,
                                          throwable: Option[Throwable],
-                                         details: String
+                                         details: String,
+                                         config: Option[ObjectNode]
```

Comment:
To Reviewers:
When exception occurs for driver when executing query, telemetry message is sent. The system config is included in the telemetry message.